docs/ThemePaletteDocs.md

```diff
@@ -0,0 +1,68 @@
+# Project Overview
+
+As part of an ongoing effort to consolidate the code base, and centralize the way we do styling, we are tackling this project to respond to several problems.
```

Comment:
Even in documentation, it's helpful to keep lines to less than 80 characters.

Reply:
Sure thing, I just fixed that